byway/src/main/webapp/index.css

```diff
@@ -0,0 +1,104 @@
+body{
+  background-image: url('/images/background1.png');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  animation: backgroundChange 18s infinite;
+}
+
+#road{
+  position: fixed;
+  left: 0px;
+  height: 60px;
+  bottom: 0px;
+  width: 100%;
+  opacity: 0.7;
+}
+
+#car{
+  position: fixed;
+  height: 300px;
+  width: 300px;
+  left: 200px;
+  bottom: -40px;
+  z-index: 2;
+  animation: suspension 1s linear infinite;
+}
+
+#tree{
+  position: fixed;
+  height: 350px;
+  width:  350px;
+  right: -200px;
+  bottom: 30px;
+  z-index: 1;
+  animation: moveRight 6s linear infinite;
+}
+
+#login-button{
+  font-size: 2vw;
+  font-weight: bold;
+  color: black;
+  text-decoration: none;
+  position: absolute;
+  left: 50%;
+  top: 40%;
+  transform: translate(-50%,-40%); 
+  background-color: white;
+  border-radius: 15%;
+  text-align: center;
+  padding: 1vw 5vw;
+}
+
+#login-button:hover{
+  opacity: 0.6
+}
+
+#title{
+  font-size: 14vw;
+  font-weight: bolder;
+  text-align: center;
+  color: #ffc05f;
+  text-shadow: 2px 2px 0 white, 2px -2px 0 white, -2px 2px 0 white, -2px -2px 0 white;
+}
+
+@keyframes backgroundChange{
+  0%{
+    background-image: url("/images/background1.png")
+  }
+  33%{
+    background-image: url("/images/background2.png")
+  }
+  67%{
+    background-image: url("/images/background3.png")
+  }
+}
+
+@keyframes moveRight{
+  0%{
+    transform: translate(0);
+    opacity: 1;
+    }
+  25%{
+    opacity: 1
+  }
+  75%{
+    transform: translateX(-3600px);
+    opacity: 0;
+  }
+  100%{
+    opacity: 0;
+  }
+}
+
+@keyframes suspension{
+  0%{
+    transform: translateY(-3px)
+  }
+  50%{
+    transform: translateY(5px)
+  }
+  100%{
+    transform: translateY(-3px)
+  }
+}
```

Comment:
small nit: add newline char at the end of files

Reply:
done